pkg/handler/processor/guesser/format_json.go

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package guesser
```

Comment:
minior nit: Do we want to break out the the implementation into a separate package

Reply:
I was doing that, then i felt like i was writing java in the 90s, then i reverted it back lol... maybe a follow up PR